model/perf.go

```diff
@@ -637,3 +638,151 @@ func (r *PerformanceResults) FindOutdatedRollups(ctx context.Context, name strin
 
 	return errors.WithStack(it.Close(ctx))
 }
+
+func GetTimeSeriesIds(ctx context.Context, db *mongo.Database) (*mongo.Cursor, error) {
```

Comment:
I don't think that the mongo.DB or mong.Cursor should be part of the public interface of the model package. Make these functions internal.

Reply:
bump